src/modules/tracker.js

```diff
@@ -1775,9 +1778,22 @@ class Tracker {
         resultId = result_id,
         result_page,
         resultPage = result_page,
+        items,
       } = parameters;
       const queryParams = {};
       const bodyParams = {};
+      let transformedItems;
```

Comment:
I believe this code is from search results where we had to support string[] and object[]. Since this is new, we don't need to support string[]. Can we copy the implementation from browse?
```
      if (items && Array.isArray(items)) {
        bodyParams.items = items.slice(0, 100).map((item) => helpers.toSnakeCaseKeys(item, false));
      }
```